### DISCUSSION TITLE
Google reCAPTCHA v3 bloque formulaire après refus des cookies

### DISCUSSION THREAD

**Discussion by jonathan-ati** [AmauriC/tarteaucitron.js#861] [Q&A]
Bonjour, 
Je rencontre une autre difficulté avec tarte au citron et le service Recaptcha. 

Lorsque j'ai un utilisateur sur le site sur ma page de création de compte avec Recaptcha V3, il refuse les cookies et le souci, c'est que l'utilisateur ne peut plus créer de compte. Le formulaire ne s'envoie plus ?! 

Est-ce que vous avez déjà rencontré le problème ? 

Si jamais vous pouvez m'aider. 
Jonathan 

PS : j'ai la version payante de TaC.

**Comment by AmauriC** [comment]
C'est le principe de recaptcha, autoriser ou non l'envoi du formulaire.
Si recaptcha n'est pas autorisé par l'internaute, le formulaire échoue.
